crates/swc_ecma_minifier/src/compress/optimize/util.rs

```diff
@@ -59,6 +59,16 @@ where
             orig_ctx,
         }
     }
+
+    pub(super) fn try_remove(&mut self, id: Id) {
```

Comment:
I don't like this solution around `ref_count` -- but there's some wired case preventing add everything inlined to removed
```js
var x,y;
x = y, y = x;
console.log(x, y, x, y)
```